crates/node/src/lib.rs

```diff
@@ -24,6 +24,12 @@ impl From<ChangedContent> for tailwindcss_oxide::ChangedContent {
 #[derive(Debug, Clone)]
 #[napi]
 pub struct ScanResult {
+  // Private information necessary for incremental rebuilds. Note: these fields are not exposed
+  // to JS
+  base: String,
+  sources: Vec<GlobEntry>,
```

Comment:
Mind = blown

Didn't know this was possible, but it keeps the information private. It's not exposed to node land.